.internal-ci/helm/fog-services/values.yaml

```diff
@@ -152,18 +158,24 @@ fogView:
 
 
 #### Fog Ledger
-fogLedgerStores:
+fogLedgerShardRangeGenerator:
+  # The dev options are set in the github workflow deployment.
+  # This is just for reference.
+  # dev:
+  #   shardSize: 25_000
+  #   exceedBlockHeightBy: 10_000
+  #   shardOverlap: 0
+  #   count: 2
```

Comment:
I don't think I understand how the override is working in .github/workflows/mobilecoin-workflow-dev-deploy.yaml, but I get the feeling it's the same as the view l
the values seem to be 10_000 for size and 5_000 for exceed by